content/influxdb/v2.0/write-data/load-data.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Load data from sources in the InfluxDB user interface (UI)
+seotitle: Load data source in UI
+list_title: Load data source in UI
+weight: 105
+description: >
+  Load data source in UI
```

Comment:
This is the summary that will appear in search engine results. We should expand the description to include popular data sources.